lib/decorators/api-bearer.decorator.ts

```diff
@@ -1,6 +1,8 @@
-import { DECORATORS } from '../constants';
-import { createMixedDecorator } from './helpers';
+import { ApiSecurity } from './api-security.decorator';
 
+/**
+ * @deprecated Use @ApiSecurity
+ */
```

Comment:
Do we really have to deprecate these decorators? (`@ApiBearerAuth()` and `@ApiOAuth2Auth()`)?

Reply:
I think it should be removed in the next version. for prevent confusion